content/hardware/04.pro/boards/portenta-x8/tutorials/waves-fleet-managment/content.md

```diff
@@ -0,0 +1,111 @@
+---
+title: 'Using FoundriesFactory Waves Fleet Management'
+description: 'Learn how to use Foundries.io Factory fleet management tool Waves to manage multiple Portenta X8 devices'
+difficulty: intermediate
+tags:
+  - Embedded Linux
+  - Flashing
+  - Foundries.io
+author: 'Benjamin Dannegård'
+hardware:
+  - hardware/04.pro/boards/portenta-x8
+---
+
+## Overview
+
+In a production environment it is convenient to plan updates, and have control over when and which devices are updated. FoundriesFactory Waves is the feature for this. It allows you to easily define a group of Portenta X8s and then push updates to that specific group. This tutorial will show you how to define that group and how to construct a Wave that can then be pushed to a group.
```

Comment:
```suggestion
In a production environment it is convenient to plan updates, and have control over when and which devices are updated. FoundriesFactory Waves is the feature for this. It allows you to easily define a group of Portenta X8s and then push updates to that specific group. This tutorial will show you how to define that group and how to construct a Wave that can then be pushed to a group.
```
Please also include information on how this differs from standard updates (waves promote targets to production by double signing them, which makes the updates more manageable/controllable).